### DISCUSSION TITLE
SharePoint Add-Ins getting retired in M365 by 2026, how does this affect or impact pnp-modern-search

### DISCUSSION THREAD

**Discussion by ezecoding** [microsoft-search/pnp-modern-search#3406] [Q&A] [answered]
Hi All,

Can you help me clarify how this announcement will affect the pnp-modern-search, because from my limited understanding of it, it's a SharePoint Online app.

https://techcommunity.microsoft.com/t5/microsoft-sharepoint-blog/sharepoint-add-in-retirement-in-microsoft-365/ba-p/3982035#:~:text=SharePoint%20Add%2DIns%20will%20stop,Clouds%20and%20Department%20of%20Defense.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/17066758/98cdbca1-bda9-4bf3-94b6-031516f48684)

I was looking at adopting it for my organisation but htis has halt me in my tracks

Thanks in advance

**Comment by kasperbolarsen** [comment, marked as answer]
See https://github.com/microsoft-search/pnp-modern-search/discussions/3388

Short version: not at all

**Reply by ezecoding** [reply]
Thanks!
